src/main/kotlin/com/nextstep/jngcii/racingcar/view/ResultView.kt

```diff
@@ -1,11 +1,26 @@
 package com.nextstep.jngcii.racingcar.view
 
-class ResultView {
-    fun printInitialPhase() {
+import com.nextstep.jngcii.racingcar.domain.History
+import com.nextstep.jngcii.racingcar.domain.Record
+
+object ResultView {
+    private const val DASH = "-"
+    private const val JOIN_SEPARATOR = ", "
+
+    fun print(history: History) {
         println("실행 결과")
+
+        history.records.forEach {
+            printSingleRecord(it)
+        }
```

Comment:
(이하 동일)
코틀린 코딩 컨벤션에서는 이렇게 chained call을 wrapping 할 때 `.` 이후에 개행하는 것을 원칙으로 하고 있습니다. 단 첫번째 요소의 경우 붙여 쓰는게 더 의미상 맞다고 판단되면 생략할 수 있습니다.
```suggestion
        history.records
            .forEach {
                printSingleRecord(it)
            }
```

https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls

Reply:
[refactor : chained call 리팩터링](https://github.com/next-step/kotlin-racingcar/pull/625/commits/c933fedfcba97fefe40895c55e1430ad0d091a8f)
오호 이런 원칙이 있었군요~! 수정했습니다!